collectors/ebpf.plugin/ebpf.c

```diff
@@ -1265,6 +1289,7 @@ static void parse_args(int argc, char **argv)
 {
     int enabled = 0;
     int disable_apps = 0;
+    int disable_cgroups = 0;
```

Comment:
WE will need to apply this to the new `shm` charts as well just merged.

Reply:
All threads without exception will have cgroup integrated. :)

Reply:
Sorry, I confused, I already applied this with last commits.